CMakeLists.txt

```diff
@@ -13,7 +13,7 @@
 cmake_minimum_required(VERSION 2.8.10 FATAL_ERROR)
 
 # explicitly set certain policies
-cmake_policy(VERSION 2.8.10)
+cmake_policy(VERSION 2.8.12)
```

Comment:
good point. In order for those changes, we need to increase the minimum to 2.8.12, which i think is safe.